### ISSUE TITLE
Getting `add_country_alias` warnings when running DuckPAN.

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/zeroclickinfo-goodies#772]
Getting `add_country_alias` warnings when running DuckPAN.

```
[02:23 PM ubuntu@ddh5 zeroclickinfo-goodies {jag/zodiac}]$ duckpan server
Checking for DuckPAN requirements...
Checking perl version...
Checking for latest App::DuckPAN...
Checking for latest DDG Perl package...
Checking for ssh...
Checking for git...
Loading Instant Answers...
add_country_alias(): alias already in use: Antigua
 at /usr/local/ddg.cpan/perl5/lib/perl5/Locale/Codes/Country.pm line 71.
add_country_alias(): alias already in use: Barbuda
 at /usr/local/ddg.cpan/perl5/lib/perl5/Locale/Codes/Country.pm line 71.
add_country_alias(): alias already in use: Russia
 at /usr/local/ddg.cpan/perl5/lib/perl5/Locale/Codes/Country.pm line 71.
add_country_alias(): alias already in use: Tobago
 at /usr/local/ddg.cpan/perl5/lib/perl5/Locale/Codes/Country.pm line 71.
add_country_alias(): alias already in use: Trinidad
 at /usr/local/ddg.cpan/perl5/lib/perl5/Locale/Codes/Country.pm line 71.
add_country_alias(): alias already in use: America
 at /usr/local/ddg.cpan/perl5/lib/perl5/Locale/Codes/Country.pm line 71.
add_country_alias(): alias already in use: Vatican
 at /usr/local/ddg.cpan/perl5/lib/perl5/Locale/Codes/Country.pm line 71.
add_country_alias(): alias already in use: US Virgin Islands
 at /usr/local/ddg.cpan/perl5/lib/perl5/Locale/Codes/Country.pm line 71.
```

**Comment by mintsoft** [comment]
@jagtalon Where are we with resolving this? I can see the PR's from @mnp that have been merged, however this issue still persists. I noticed that it seemed to cause some confusion here: https://github.com/duckduckgo/duckduckgo-documentation/issues/230

**Comment by MrChrisW** [comment]
@mintsoft I don't think any of @mnp PR's have been merged, it looks like https://github.com/duckduckgo/duckduckgo/pull/108 is the main PR. Not sure who it's waiting on.

**Comment by moollaza** [comment]
@jagtalon , correct me if I'm wrong, but AFAIK, we're working on exposing a new function that will take a string (candidate location) and give us a place object which we can use to determine the city/country, etc. for a given place. Once that's ready I think we'll be able to replace `the Locale::Code` lib?

Alternatively as an easy (but ugly) interim fix, we could simply remove the duplicates and consolidate the aliases in one Goodie for now?

It looks like we were expecting changes to be made to https://github.com/duckduckgo/duckduckgo/pull/108

**Comment by mnp** [comment]
Yes, Ugly interim fix for #108 was me until life had other plans :-)  It
was exactly that, factoring out to one goodie and remove dupes.  I'll try
to finish that up.

On Tue, Mar 31, 2015 at 11:55 AM, Zaahir Moolla <notifications@github.com>
wrote:

> @jagtalon <https://github.com/jagtalon> , correct me if I'm wrong, but
> AFAIK, we're working on exposing a new function that will take a string
> (candidate location) and give us a place object which we can use to
> determine the city/country, etc. for a given place. Once that's ready I
> think we'll be able to replace the Locale::Code lib?
>
> Alternatively as an easy (but ugly) interim fix, we could simply remove
> the duplicates and consolidate the aliases in one Goodie for now?
>
> It looks like we were expecting changes to be made to
> duckduckgo/duckduckgo#108
> <https://github.com/duckduckgo/duckduckgo/pull/108>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/duckduckgo/zeroclickinfo-goodies/issues/772#issuecomment-88141279>
> .
>



-- 
Mitchell Perilstein <mitchell.perilstein@gmail.com>
(484) 876-1444

**Comment by moollaza** [comment]
Closing. This was fixed a while ago :) We now defined all aliases in one place: https://github.com/duckduckgo/duckduckgo/blob/master/lib/DDG/Meta/CountryCodes.pm
